src/PiWeb.Sdk.Import/PiWeb.Sdk.Import.csproj

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
     <Nullable>enable</Nullable>
-    <AssemblyName>Zeiss.PiWeb.Import.Sdk</AssemblyName>
-    <RootNamespace>$(AssemblyName)</RootNamespace>
+    <AssemblyName>Zeiss.PiWeb.Sdk.Import</AssemblyName>

```

Comment:
This will also affect the NuGet name, but it guess this is expected. 
We will just need to update all the references to the new name.

Reply:
Yes this is expected and accepted.